DH_Interface/Classes/DHDeployMenu.uc

```diff
@@ -1720,15 +1720,16 @@ function UpdateSquads()
             else
             {
                 C.i_LockSquad.Image = UnlockIcon;
-                C.b_LockSquad.SetHint(default.LockText);
 
                 if (bCanSquadBeLocked)
                 {
                     C.i_LockSquad.ImageColor.A = 255;
+                    C.b_LockSquad.SetHint(default.LockText);
                 }
                 else
                 {
-                    C.i_LockSquad.ImageColor.A = 128;
+                    C.i_LockSquad.ImageColor.A = 64;
+                    C.b_LockSquad.SetHint("Squad can be locked only if it has " $ SRI.SquadLockMemberCountMin $ " or more members");
```

Comment:
You can use `@` to concatenate strings with a space.

Reply:
I know but its... weird.